subprojects/dependency-management/dependency-management.gradle.kts

```diff
@@ -0,0 +1,63 @@
+import org.gradle.build.ClasspathManifest
+import org.gradle.gradlebuild.testing.integrationtests.cleanup.EmptyDirectoryCheck
+import org.gradle.gradlebuild.unittestandcompile.ModuleType
+
+plugins {
+    `java-library`
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_6
+}
+
+dependencies {
+    api(project(":core"))
+    api(project(":versionControl"))
+
+    implementation(project(":resources"))
+    implementation(project(":resourcesHttp"))
+
+    implementation(library("asm"))
+    implementation(library("asm_commons"))
+    implementation(library("asm_util"))
+    implementation(library("commons_lang"))
+    implementation(library("commons_io"))
+    implementation(library("ivy"))
+    implementation(library("slf4j_api"))
+    implementation(library("gson"))
+    implementation(library("jcip"))
+    implementation(library("maven3"))
+
+    runtimeOnly(library("bouncycastle_provider"))
+    runtimeOnly(project(":installationBeacon"))
+
+    testImplementation(library("nekohtml"))
+
+    integTestRuntimeOnly(project(":ivy"))
+    integTestRuntimeOnly(project(":maven"))
+    integTestRuntimeOnly(project(":resourcesS3"))
+    integTestRuntimeOnly(project(":resourcesSftp"))
+    integTestRuntimeOnly(project(":testKit"))
+
+    testFixturesImplementation(project(":internalIntegTesting"))
+}
+
+gradlebuildJava {
+    // TODO Source Compatibility was set to 1.6. Why? Is this worker code?
+    moduleType = ModuleType.WORKER
+}
+
+testFixtures {
+    from(":core")
+    from(":messaging")
+    from(":modelCore")
+    from(":versionControl")
+}
+
+val verifyTestFilesCleanup by tasks.getting(EmptyDirectoryCheck::class) {
```

Comment:
:x: should use the `testFilesCleanup` extension instead:
```kotlin
testFilesCleanup {
    isErrorWhenNotEmpty = false
}
```

Reply:
Wasn't aware of that. Thanks. Changed it.